ale/base/data_naif.py

```diff
@@ -68,6 +68,14 @@ def ikid(self):
     def spacecraft_id(self):
         return spice.bods2c(self.spacecraft_name)
 
+    @property
+    def target_id(self):
+        return spice.bods2c(self.target_name)
+
+    @property
+    def body_frame_code(self):
+        return spice.gipool('BODY_FRAME_CODE', 0, 1)
```

Comment:
This is not actually available as a kernel variable. `BODY_FRAME_CODE` is one of those values ISIS creates. You can get the reference frame for a body via [cidfrm](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/cidfrm_c.html).

Reply:
Ok, so I see where they are using cidfrm_c() in Spice.cpp in the init, but it looks like another path (the else block - which seems to be saying if you are not using naif...?) ends up calling getInteger() with 'BODY_FRAME_CODE' as it's key, which eventually makes use of the gipool_c() method, so I am a little confused here...

Reply:
The "using naif" flag is for if you have SPICE data attached or not. So, anything that depends on that is for post spiceinit, I.E. post ALE. Also, getInteger doesn't always use gipool_c(). If the value is already stored in the NaifKeywords object it doesn't use gipool_c().